packages/@webex/plugin-meetings/src/constants.ts

```diff
@@ -70,6 +70,8 @@ export const TYPE = 'type';
 export const VIDEO = 'video';
 export const VIDEO_STATUS = 'videoStatus';
 
+export const ENGLISH_LANGUAGE = 'en';
```

Comment:
Could we change this to LANGUAGE_ENGLISH instead so that it's only the second word of the constant that changes?

Reply:
Sure, I can change and we have ENGLISH_LANGUAGE only here because it is default. So, not expecting other constants be added here